wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/unterbrechungsuhrzeit/UnterbrechungsUhrzeitControllerIntegrationTest.java

```diff
@@ -0,0 +1,212 @@
+        )
+        void newDataIsSaved() throws Exception {
+            val wahlbezirkID = "wahlbezirkID";
+            val writeDto = new UnterbrechungsUhrzeitWriteDTO(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            val request = buildPostRequest(wahlbezirkID, writeDto);
+
+            mockMvc.perform(request).andExpect(status().isCreated());
+
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo).isEqualTo(expectedUnterbrechungsUhrzeit);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void existingDataIsOverwritten() throws Exception {
+
+            val wahlbezirkID = "wahlbezirkID";
+            val writeDto1 = new UnterbrechungsUhrzeitWriteDTO(LocalDateTime.of(2023, 1, 1, 12, 0, 0));
+            val request1 = buildPostRequest(wahlbezirkID, writeDto1);
+
+            mockMvc.perform(request1).andExpect(status().isCreated());
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo1 = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit1 = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto1));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo1).isEqualTo(expectedUnterbrechungsUhrzeit1);
+
+            val writeDto2 = new UnterbrechungsUhrzeitWriteDTO(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            val request2 = buildPostRequest(wahlbezirkID, writeDto2);
+
+            mockMvc.perform(request2).andExpect(status().isCreated());
+
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo2 = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit2 = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto2));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo2).isEqualTo(expectedUnterbrechungsUhrzeit2);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
```

Comment:
Das Read-Recht wird erst nach dem mockMVC.perform benötigt und dort wirt es auch explizit gesetzt
```suggestion
        @WithMockUser(
                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT }
        )
```